bokeh/core/property/container.py

```diff
@@ -356,6 +356,22 @@ def __init__(self, default={}, help=None):
     def __str__(self):
         return self.__class__.__name__
 
+class RestrictedDict(Dict):
+    """ Check for disallowed key(s).
+
+    """
+
```

Comment:
@cyborg1995 you will need an initializer here to peel off `disallow` and set `self.disallow`